src/main/java/hw6/composite/SidebarComposite.java

```diff
@@ -0,0 +1,37 @@
+package hw6.composite;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SidebarComposite extends AbstractPageComposite {
+    @FindBy(css = ".sidebar-menu > li")
+    private List<WebElement> leftSidebarElements;
+
+    @FindBy(linkText = "Service")
+    private WebElement serviceButton;
+
+    @FindBy(linkText = "Different elements")
+    private WebElement differentElementsButton;
+
+    @FindBy(linkText = "User Table")
+    private WebElement userTableButton;
```

Comment:
These items could be combined to the `List<WebElement>`

Reply:
done